examples/i2c_reply_datum/i2c_reply_datum.ino

```diff
@@ -0,0 +1,33 @@
+#include "I2CTransfer.h"
+
+
+I2CTransfer myTransfer;
+
+struct __attribute__((packed)) STRUCT {
+  char z;
+  float y;
+} testStruct;
+
+void I2C_RequestHandler(void)
+{
+  myTransfer.replyWithDatum(testStruct);
```

Comment:
@tommag I think it would be good to double check that the requested data length matches the struct the sender wants to report. There should be a check for this either at the sketch or lib level. Also, it might be nice to request a specific packet type (Packet ID field)